packages/maskbook/src/_locales/en/messages.json

```diff
@@ -315,10 +315,19 @@
     "plugin_wallet_on_connect": "Connect Wallet",
     "plugin_wallet_wrong_network": "Wrong Network",
     "plugin_wallet_setup_title_create": "Create a Wallet",
+    "plugin_wallet_create_import_choose": "Create Wallet & Import Wallet",
+    "plugin_wallet_create_a_new_wallet": "Create a New Wallet",
+    "plugin_wallet_create_a_new_wallet_description": "Mask network supports Eth and Polka networks ",
+    "plugin_wallet_import_wallet": "Import the Wallet",
+    "plugin_wallet_import_wallet_description": "Mask network supports Private Key、JSON.File and Mnemonic words.",
```

Comment:
```suggestion
    "plugin_wallet_import_wallet_description": "Mask network supports Private Key, JSON File, and Mnemonic words.",
```

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/3298/commits/ceb17bf35a934b93a06b2c82020d6bb7e973e963
Removed `JSON.File` by the way, since it's not ready yet.